pom.xml

```
@@ -238,7 +237,7 @@
                                     <version>3.0.3</version>
                                 </requireMavenVersion>
                                 <requireJavaVersion>
-                                    <version>[1.8,9)</version>
+                                    <version>[8,21)</version>
```

```suggestion
                                    <version>[17,22)</version>
```
We should limit this to 17, and allow 21, right?

---

Yes, correct.